packages/eslint-plugin-js/rules/linebreak-style/linebreak-style.ts

```diff
@@ -3,14 +3,21 @@
  * @author Erik Mueller
  */
 
+import type { TSESLint, TSESTree } from '@typescript-eslint/utils'
 import { createGlobalLinebreakMatcher } from '../../utils/ast-utils'
+import { createRule } from '../../utils/createRule'
 
 // ------------------------------------------------------------------------------
 // Rule Definition
 // ------------------------------------------------------------------------------
 
-/** @type {import('eslint').Rule.RuleModule} */
-export default {
+type RuleOptions = ['unix' | 'windows']
+
+type MessageIds =
+  | 'expectedLF'
+  | 'expectedCRLF'
```

Comment:
Oh I mean you can import them from `./type`

Reply:
Oops, fixed it